src/component/feature/CopyFeature/CopyFeature.tsx

```diff
@@ -114,7 +114,6 @@ export const CopyFeatureToggle = () => {
                         variant="outlined"
                         size="small"
                         inputRef={inputRef}
-                        required
```

Comment:
Could perhaps replace `required` with `aria-required` to retain some semantics.

Looks like the removal allows the form to be submitted if the field is never focused.

Reply:
Would something like https://github.com/Unleash/unleash-frontend/pull/1117/commits/8edd7066cc09df47c6d4768ada9667fce2b16afb make more sense?